holidays/countries/suriname.py

```diff
@@ -0,0 +1,258 @@
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=SurinameIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_WORKDAY)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        dt = self._add_new_years_day(tr("Nieuwjaarsdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if 1981 <= self._year <= 1992 or 2012 <= self._year <= 2020:
+            # Day of Liberation and Renewal.
+            self._add_holiday_feb_25(tr("Dag van Bevrijding en Venieuwing"))
+
+        # Good Friday.
+        dt = self._add_good_friday(tr("Goede Vrijdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Easter Monday.
+        dt = self._add_easter_monday(tr("Tweede Paasdag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year <= 1975:
+            # Birthday of H.M. the Queen.
+            self._add_holiday_apr_30(tr("Verjaardag van H.M. de Koningin"))
+
+        # Labor Day.
+        dt = self._add_labor_day(tr("Dag van de Arbeid"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        # Day of Freedoms.
+        dt = self._add_holiday_jul_1(
+            # Day of Freedoms.
+            tr("Keti Koti Dey") if 2008 <= self._year <= 2024 else tr("Dag der Vrijheden")
+        )
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year >= 2007:
+            # Indigenous People Day.
+            dt = self._add_holiday_aug_9(tr("Dag der Inheemsen"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Day of the Maroons.
+            dt = self._add_holiday_oct_10(tr("Dag der Marrons"))
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        if self._year >= 1976:
+            name = (
+                # Independence Day.
+                tr("Onafhankelijkheidsdag")
+                if self._year >= 2008
+                # Republic Day.
+                else tr("Dag van de Republiek")
+            )
+            dt = self._add_holiday_nov_25(name)
+            if self._year > 2024:
+                self._add_observed(dt)
+
+        # Christmas Day.
+        dt = self._add_christmas_day(tr("Eerste Kerstdag"))
+        if self._year >= 2024:
+            self._add_observed(dt)
+
+        # Second Day of Christmas.
+        dt = self._add_christmas_day_two(tr("Tweede Kerstdag"))
+        if self._year >= 2024:
+            self._add_observed(dt)
+
+        # Holi.
+        dt = self._add_holi(tr("Holi-dag"))
+        if self._year > 2024:
+            self._add_observed(dt)
+
+        if self._year >= 2012:
+            # Diwali.
+            dt = self._add_diwali(tr("Divali dag"))
```

Comment:
In latest edition it's "Holi-Phagwa" and "Divali", in previous - "Holi-dag" and "Divali", before that - "Holi-dag" and "Divali dag"... They are not very careful with official names. Well, let's use the last one ("Holi-Phagwa" and "Divali")